docs/user/alerting/action-types/index.asciidoc

```diff
@@ -54,3 +54,40 @@ Execution time field::  This field will be automatically set to the time the ale
 Index actions have the following properties:
 
 Document::  The document to index in json format.
+
+The create test index using index API:
```

Comment:
I like having this section, but feels like it should be at the end of the doc for this action type - seems like the flow works better if we show them the example document they'd use in the alert, and then have a section at the bottom with this example of a creating an index to use with the action.

Reply:
I agree